### ISSUE TITLE
xbox controller drifting 

### ISSUE THREAD

**Issue by fvaltrock** [Ryujinx/Ryujinx-Games-List#471]
#### 1.0.0

Game works fine.
```
Last error returned.
e.g.: No error. It is about the Controller
```

I changed the drifting value in the Config from .05 to .5 but my Character still moves always to the right.

#### Log file :

I dont think there is any log neccesary but i will post everything that is needed.

**Comment by fvaltrock** [comment]
PS: The Controller is Brand New and never had any problems so i am pretty shure that my controller is fine.

**Comment by AcK77** [comment]
Closed in favor of #250!
